instructions.json

```diff
@@ -93,7 +93,27 @@
             "straight": "Go straight[ onto {way_name}]"
         },
         "use lane": {
-            "defaultInstruction": "Continue {modifier}"
+            "defaultInstruction": "{laneInstruction} and continue straight[ onto {way_name}]",
+            "left": "{laneInstruction} and turn left[ onto {way_name}]",
```

Comment:
The conjunction here should be "to", not "and". "And" when spoken tends to make the user think there are two different steps, and that would get even more confusion when linking instructions together.